.eslintrc.js

```diff
@@ -17,5 +17,14 @@ module.exports = {
         "@typescript-eslint/no-unsafe-member-access": "off",
       },
     },
+    {
+      files: ["examples/**/*"],
```

Comment:
Is this the best way to do this?

Reply:
I think there are ways we could accomplish this with separate eslint configs maybe but I don't think it's worth it. This works for me 👍🏻